### ISSUE TITLE
Pre-selected PFS and not enough utility token to pay

### ISSUE THREAD

**Issue by christianbrb** [raiden-network/light-client#697]
## Description
We need to decide what to do if there is a pre-selected paid PFS and the user does not have enough utility tokens to pay it.

## Acceptance criteria
- 

## Tasks
- [ ]

**Comment by andrevmatos** [comment]
I don't think there's anything to do here. The pre-selected PFS case (`config.pfs`) should only skip the confirmation dialog fee **iff it's free**. If not, the user is still prompted with the single PFS instance, can review & confirm the fee, can deposit more if not enough, or will get the proper error if don't.

**Comment by christianbrb** [comment]
@kelsos Would it already work the way @andrevmatos described?

**Comment by kelsos** [comment]
@christianbrb It seems that I forgot about this change completely. Initially I would check for a set PFS in the configuration directly, but when we added the skip functionality this was moved to the 'findPFS` method on the SDK. Looking at the dApp code the only change the Service selection will be skipped is if the PFS is free and you don't need to pay for the route. Otherwise, the dialog will be visible for you to select.
